css/components-list.css

```diff
@@ -0,0 +1,124 @@
+@import url("https://fonts.googleapis.com/css2?family=PT+Sans+Narrow:wght@400;700&display=swap");
+@font-face {
+  font-family: Jurassic Park;
+  src: url(JurassicPark-BL48.ttf);
+}
+
+:root {
+  --primary-color: #00919c;
+  --title-color: #cccccc;
+  --subtitle-color: #808080;
+  --card-bg-color: #27272785;
+  --background-color: #101212;
+}
+
+html,
+body,
+h1,
+p,
```

Comment:
don't reset tags as it changes the purpose of them

Reply:
These reset tags are just restricted to this landing page. Wouldn’t be of much trouble elsewhere.